### DISCUSSION TITLE
MCCGQ12LM cannot be configured

### DISCUSSION THREAD

**Discussion by jmachta** [Koenkk/zigbee2mqtt#19399] [Devices]
Recently I have obtained a MCCGQ12LM sensor from Xiaomi and I see strange behavior. The device gets paired, but never configured - when I press the configure button in web ui, I get a message that "Device cannot be configured". There does not seems to be anything wrong in log, however perhaps I do not know where to look. Below is the relevant snippet of messages from z2m. 

Interesting is that when I downgraded z2m to version 1.22.0, device paired and configured and all data from it is reported. I am using sonoff zigbee 3.0 usb dongle plus... Then I have upgraded back to 1.33.1 and I see only LQI value, nothing else, other values seem to be stale. Unfortunately some of my devices need new version of z2m so I cannot keep the version down... I would also like to avoid dongle upgrade, but if necessary, I would have to risk it. 

Any ideas are welcome. 
`
Info 2023-10-23 22:49:04Device '0x54ef4410008503b0' is supported, identified as: Xiaomi Aqara T1 door & window contact sensor (MCCGQ12LM)
Info 2023-10-23 22:49:05MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"definition":{"description":"Aqara T1 door & window contact sensor","exposes":[{"access":1,"description":"Indicates if the contact is closed (= true) or open (= false)","label":"Contact","name":"contact","property":"contact","type":"binary","value_off":true,"value_on":false},{"access":1,"description":"Remaining battery in %, can take up to 24 hours before reported.","label":"Battery","name":"battery","property":"battery","type":"numeric","unit":"%","value_max":100,"value_min":0},{"access":1,"description":"Voltage of the battery in millivolts","label":"Voltage","name":"voltage","property":"voltage","type":"numeric","unit":"mV"},{"access":1,"description":"Link quality (signal strength)","label":"Linkquality","name":"linkquality","property":"linkquality","type":"numeric","unit":"lqi","value_max":255,"value_min":0}],"model":"MCCGQ12LM","options":[],"supports_ota":true,"vendor":"Xiaomi"},"friendly_name":"0x54ef4410008503b0","ieee_address":"0x54ef4410008503b0","status":"successful","supported":true},"type":"device_interview"}'`

**Comment by TautrimasM** [comment]
Hi, I am facing a similar issue and trying to troubleshoot.
Are you running zigbee2mqtt on a linux machine? Are you using an external converter? have you tried running it on windows?
